frontend/src/authentication/components/register/RegisterPage.jsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Field, reduxForm } from 'redux-form';
+import { registerUser } from './../../containers/registerUser.js';
+
+const form = reduxForm({
+    form: 'register',
+});
+
+class RegisterPage extends React.Component {
+    handleFormSubmit(formProps) {
+        this.props.registerUser(formProps);
+    }
+
+    renderAlert() {
+        if (this.props.errorMessage) {
+            return (
+                <div>
+                    <span><strong>Error!</strong> {this.props.errorMessage}</span>
+                </div>
+            );
+        } else {
+            return null;
+        }
+    }
+
+    render() {
+        const { handleSubmit } = this.props;
+
+        return (
+            <form onSubmit={handleSubmit(this.handleFormSubmit.bind(this))}>
+                {this.renderAlert()}
+                <div>
+                    <div>
+                        <Field name='firstName' component='input' type='text' />
+                    </div>
+                    <div>
+                        <Field name='lastName' component='input' type='text' />
+                    </div>
+                </div>
+                <div>
+                    <div>
+                        <Field name='email' component='input' placeholder='Email' type='text' />
+                    </div>
+                </div>
+                <div>
+                    <div>
+                        <Field name='password' component='input' placeholder='Password' type='password' />
```

Comment:
How to adjust validation error? E.g. change color or position? I haven't checked UI but I assume it will be different from what you see on Sign In page. 
1. UI should be consistent and implemented keeping DRY principles in mind
2. UI should be flexible enough to adjust for changing requirements (e.g. change color, message, position etc)